package.json

```diff
@@ -11,7 +11,7 @@
     "pretest": "npm run lint",
     "lint": "eslint src __tests__",
     "test": "jest --coverage",
-    "test:watch": "jest --watch --testPathIgnorePatterns integration"
+    "test:watch": "jest --watch"
```

Comment:
did you intend to leave this change in place on commit?

Reply:
I did! In a previous PR I re-wrote the integration tests. as a result they can now be ran watch mode without problem